contracts/voting/VotingStructs.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache 2.0
+// Copyright © 2021 Anton "BaldyAsh" Grigorev. All rights reserved.
+pragma solidity ^0.8.0;
+
+enum Building {
+    Empty,
+    University,
+    Hospital,
+    Bank,
+    Factory,
+    Stadium,
+    Monument
+}
+
+struct Region {
+    bool active;
+    uint256 startVotingTimestamp;
+}
+
+struct City {
+    uint256 regionId;
+    string name;
+    uint256 population;
+    uint256 votePrice;
+    bool active;
+}
+
+struct NewCity {
+    string name;
+    uint256 population;
+    uint256 votePrice;
+}
+
+struct Nominee {
+    uint256 mayorId;
+    uint256 votes;
+}
+
+struct BuildingInfo {
+    Building building;
+    uint256 season;
+}
+
+struct ClaimInfo {
+    uint256 cityId;
+    uint256[] seasonIds;
+    Building[] bulidings;
```

Comment:
bulidings -> buildings

Reply:
also why is part of the claim? can't we just go through possible buildings checking if they are built?

Reply:
Removed "buildings" as arguments of public methods